docs/cloud-native-security/cspm-get-started.asciidoc

```diff
@@ -0,0 +1,124 @@
+[[cspm-get-started]]
+= Get started with CSPM
+
+[discrete]
+[[cspm-overview]]
+== Overview
+
+This page explains how to get started monitoring the security posture of your cloud assets using the Cloud Security Posture Management (CSPM) integration.
+
+NOTE: This integration currently supports Amazon Web Service (AWS). Support for other major cloud service providers is coming soon.
+
+.Requirements
+[sidebar]
+--
+* The CSPM integration is available to all Elastic Cloud users. For on-prem deployments, it requires an https://www.elastic.co/pricing[Enterprise subscription].
+* To view posture data, you need `read` privileges for the following {es} indices: `logs-cloud_security_posture.findings_latest-*`, `logs-cloud_security_posture.scores-*`, `Logs-cloud_security_posture.findings`.
+* The user who gives the CSPM integration AWS permissions needs to be an AWS account `admin` user.
+--
+
+[discrete]
+[[cspm-setup]]
+== Set up CSPM for AWS
+
+[discrete]
+[[cspm-add-and-name-integration]]
+=== Add and name your CSPM integration
```

Comment:
Creating a name doesn't seem like such a central or unique part of this process that it needs to be mentioned in the heading; just "add the integration" seems enough to capture the overall step in the process.
```suggestion
=== Add your CSPM integration
```